### ISSUE TITLE
Build error on mac

### ISSUE THREAD

**Issue by nmittler** [google/quiche#33]
I get the following error when running `bazel build //quiche:quiche_core`:

```
ERROR: /private/var/tmp/_bazel_nathanmittler/1814428b3f255bb79b4de47698a8348b/external/com_google_googleurl/url/BUILD:12:11: Compiling url/url_idna_icu.cc failed: (Exit 1): wrapped_clang_pp failed: error executing command (from target @com_google_googleurl//url:url) external/local_config_cc/wrapped_clang_pp '-D_FORTIFY_SOURCE=1' -fstack-protector -fcolor-diagnostics -Wall -Wthread-safety -Wself-assign -fno-omit-frame-pointer -O0 -DDEBUG '-std=c++11' ... (remaining 36 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
external/com_google_googleurl/url/url_idna_icu.cc:83:25: error: no matching function for call to 'uidna_nameToASCII'
    int output_length = uidna_nameToASCII(uidna, src, src_len, output->data(),
                        ^~~~~~~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.1.sdk/usr/include/unicode/uidna.h:294:1: note: candidate function not viable: no known conversion from 'const char16_t *' to 'const UChar *' (aka 'const unsigned short *') for 2nd argument
uidna_nameToASCII(const UIDNA *idna,
^
```

I'm building on an m1 macbookpro with OSX Ventura (13.1) and the latest Xcode SDK (13.1).

Anyone seen this before or have a workaround?

**Comment by vasilvv** [comment]
Building with `--@com_google_googleurl//build_config:system_icu=0` should fix the issue.

**Comment by nmittler** [comment]
@vasilvv ah perfect - that seems to do the trick! ... thanks for the quick reply.

Is there any action here that might help others going forward? Maybe an update to the readme?

**Comment by DavidSchinazi** [comment]
Fixed by 0c75f987990bfb2fe27eeaa8f3cc78f98f3ef42d - now `bazel build //quiche:quiche_core` works on macOS
